debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java

```diff
@@ -228,16 +231,23 @@ public void readOnlyApplicationIntent() throws Exception {
         // Verify that multiple subsequent transactions are used in streaming phase with read-only intent
         try (final SqlServerConnection admin = TestHelper.adminConnection()) {
             final Set<Long> txIds = new HashSet<>();
-            Awaitility.await().atMost(TestHelper.waitTimeForRecords() * 5, TimeUnit.SECONDS).pollInterval(100, TimeUnit.MILLISECONDS).until(() -> {
-                admin.query(
-                        "SELECT (SELECT transaction_id FROM sys.dm_tran_session_transactions AS t WHERE s.session_id=t.session_id) FROM sys.dm_exec_sessions AS s WHERE program_name='"
-                                + appId + "'",
-                        rs -> {
-                            rs.next();
-                            txIds.add(rs.getLong(1));
-                        });
-                return txIds.size() > 2;
-            });
+            try {
+                Awaitility.await().atMost(TestHelper.waitTimeForRecords() * 5, TimeUnit.SECONDS).pollInterval(100, TimeUnit.MILLISECONDS).until(() -> {
+                    admin.query(
+                            "SELECT (SELECT transaction_id FROM sys.dm_tran_session_transactions AS t WHERE s.session_id=t.session_id) FROM sys.dm_exec_sessions AS s WHERE program_name='"
+                                    + appId + "'",
+                            rs -> {
+                                rs.next();
+                                txIds.add(rs.getLong(1));
+                            });
+                    return txIds.size() > 2;
+                });
+            }
+            catch (ConditionTimeoutException e) {
+                System.out.println("Transaction Ids: " + txIds);
```

Comment:
Please replase `System.out` calls

Reply:
My apologies, was too quick to push.  Fixed & only the applicable changes are now in the test case.